examples/FeatureFlagDemo/ClaimTypes.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace FeatureFlagDemo
+{
+    static class ClaimTypes
+    {
+        public static string GroupName = "http://schemas.featureflagdemo.featuremanagement.microsoft.com/claims/groupname";
```

Comment:
>microsoft [](start = 91, length = 9)

we might not want to use microsoft.com here since user would just copy it

Reply:
That should be fine. A look at [`System.Security.Claims.ClaimTypes`](https://docs.microsoft.com/en-us/dotnet/api/system.security.claims.claimtypes?view=netframework-4.8) shows plenty of claims that use microsoft.com. The only problem is there is no group name claim.